RadixWallet/Core/SharedModels/Assets/FiatWorth.swift

```diff
@@ -145,12 +145,3 @@ extension FiatWorth {
 		return attributedString
 	}
 }
-
-extension FiatCurrency {
```

Comment:
moved to https://github.com/radixdlt/babylon-wallet-ios/blob/sargon_profile_integration_stage_1_batch_1/RadixWallet/ProfileSargonMigration/Stage1/Stage1MigrateToSargon%2BFiatCurrency.swift#L5